coreplugins/botadmin.py

```diff
@@ -45,12 +45,16 @@ def default_config(self, container=None):
             'max_dump': 4  # maximum storage/configdump messages to show
         }
 
-    @commands.command(name="subsys", help="Shows registrations on subsystems",
-                      description="Shows registrations on subsystems. If a subsystem name is given, "
-                                  "only registrations for this subsystem will be shown.",
+    @commands.command(name="subsys", hidden=True)
+    async def cmd_subsys(self, ctx):
+        await ctx.send("Did you mean: `!service`")
```

Comment:
Execute !service automatically after notice that !subsys is depraced and will be removed in future version